src/styles/main.scss

```diff
@@ -0,0 +1,284 @@
+        background: $c-seconds;
+        box-shadow: 0 0 #{$size-glow}vw #{$size-glow-blur}vw $c-sec-glow;
+      }
+
+      &::after {
+        content: "";
+        position: absolute;
+        top: 48%;
+        left: 48%;
+        width: 4%;
+        height: 4%;
+        border-radius: 50%;
+        background: $c-sec-axis;
+        box-shadow: 0 0 #{$size-glow}vw #{$size-glow-blur}vw $c-sec-glow;
+      }
+    }
+  }
+
+  &__minutes {
+    @media (orientation: landscape) {
+      width: #{$size-minute}vh;
+      height: #{$size-minute}vh;
+    }
+    @media (orientation: portrait) {
+      width: #{$size-minute}vw;
+      height: #{$size-minute}vw;
+    }
+    position: absolute;
+    z-index: 2;
+
+    @keyframes minuting {
+      0% {
+        transform: rotate(0);
+      }
+      @for $step from 1 through 60 {
+
+        #{$step * 10/6}% {
+          transform: rotate(#{($step) * 6}deg);
+        }
+      }
+    }
+
+    &-minute {
+      position: absolute;
+
+      width: 100%;
+      height: 100%;
+
+      animation-name: minuting;
+      animation-duration: 3600s;
+      animation-iteration-count: infinite;
+      animation-delay: var(--minutes);
+
+      &::before {
+        border-radius: $size-br-radius;
+
+        content: "";
+        position: absolute;
+        top: 5%;
+        left: 48.5%;
+        width: 3%;
+        height: 45%;
+        background: $c-minutes;
+        box-shadow: 0 0 #{$size-glow}vw #{$size-glow-blur}vw $c-min-glow;
+      }
+
+      &::after {
+        content: "";
+        position: absolute;
+        top: 47%;
+        left: 47%;
+        width: 6%;
+        height: 6%;
+        border-radius: 50%;
+        background: $c-min-axis;
+        box-shadow: 0 0 #{$size-glow}vw #{$size-glow-blur}vw $c-min-glow;
+      }
+    }
+  }
+
+  &__hours {
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    z-index: 1;
+
+    @keyframes houring {
+      0% {
+        transform: rotate(0);
+      }
+      @for $step from 1 through 60 {
+
+        #{$step * 10/6}% {
+          transform: rotate(#{($step) * 6}deg);
+        }
+      }
+    }
+
+    &-hour {
+      position: absolute;
+
+      width: 100%;
+      height: 100%;
+
+      animation-name: houring;
+      animation-duration: 43200s;
+      animation-iteration-count: infinite;
+      animation-delay: var(--hours);
+
+      &::before {
+        border-radius: $size-br-radius;
+
+        content: "";
+        position: absolute;
+        top: 9.5%;
+        left: 47.75%;
+        width: 4.5%;
+        height: 40.5%;
+        background: $c-hours;
+        box-shadow: 0 0 #{$size-glow}vw #{$size-glow-blur}vw $c-hour-glow;
+      }
+
+      &::after {
+        content: "";
+        position: absolute;
+        top: 45.5%;
+        left: 45.5%;
+        width: 9%;
+        height: 9%;
+        border-radius: 50%;
+        background: $c-hour-axis;
+        box-shadow: 0 0 #{$size-glow}vw #{$size-glow-blur}vw $c-hour-glow;
+      }
+    }
+  }
+}
```

Comment:
A lot of repeated code for different stopwatch elements. Try to write as little code as possible. Create mixins, placeholders, functions, loops, etc. to achieve this.

Reply:
trying to be smarter and keep it more readable and reusable
thank you